src/renderer/runner.ts

```diff
@@ -21,6 +27,12 @@ export enum ForgeCommands {
   MAKE = 'make',
 }
 
+const resultString: Record<RunResult, string> = Object.seal({
```

Comment:
nit: Is there a preference for `Object.seal` over `Object.freeze` here?

Reply:
Good nit. Using `Object.freeze()` here is better.

Reply:
Changed in b72dffc2183adaa8cf9b1e323219a7505f2cb6c9